internal/pkg/describe/status.go

```diff
@@ -341,6 +335,395 @@ func (a *apprunnerServiceStatus) HumanString() string {
+
+	fmt.Fprint(writer, color.Bold.Sprint("Task Summary\n\n"))
+	writer.Flush()
+
+	// Write summary of running tasks.
+	header := "Running"
+	bar := summaryBar(
+		[]int{
+			(int)(s.Service.RunningCount),
+			(int)(s.Service.DesiredCount) - (int)(s.Service.RunningCount),
+		},
+		[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+	stringSummary := fmt.Sprintf("%d/%d Desired Tasks Running", s.Service.RunningCount, s.Service.DesiredCount)
+	fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+	// Write summary of task definition revision.
+	desiredTaskDefVersion, err := awsECS.TaskDefinitionVersion(s.Service.TaskDefinition)
+	if err == nil && s.Service.RunningCount > 0 {
+		header := "Deployment"
+		data := s.taskDefinitionRevisionData()
+		bar := summaryBar(
+			[]int{
+				data[desiredTaskDefVersion],
+				(int)(s.Service.RunningCount) - data[desiredTaskDefVersion],
+			},
+			[]string{color.Green.Sprint("■"), color.Yellow.Sprint("□")})
+		stringSummary := fmt.Sprintf("%d/%d Running task definition version %d (desired)",
+			data[desiredTaskDefVersion],
+			s.Service.RunningCount,
+			desiredTaskDefVersion)
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+	}
+
+	// Write summary of HTTP health and container health.
+	if s.shouldShowHealthSummary() {
+		header := "Healthy"
+		if s.shouldShowHTTPHealth() {
+			healthyCount := s.healthyHTTPTasksCount()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes HTTP Health Checks", healthyCount, s.Service.RunningCount)
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+			header = ""
+		}
+
+		if s.shouldShowContainerHealth() {
+			healthyCount, _, _ := s.containerHealthData()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d Passes Container Health Checks", healthyCount, s.Service.RunningCount)
+
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+		}
+	}
+
+	// Write summary of capacity providers.
+	if s.shouldShowCapacityProvider() {
+		header := "Capacity Provider"
+		fargate, spot, unset := s.capacityProviderData()
+		bar := summaryBar([]int{fargate, spot, unset}, []string{color.Blue.Sprint("f"), color.Magenta.Sprint("s"), color.Grey.Sprint("d")})
+		var cpSummaries []string
+		if fargate != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate", fargate, s.Service.RunningCount))
+		}
+		if spot != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate Spot", spot, s.Service.RunningCount))
+		}
+		if unset != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on default capacity provider", unset, s.Service.RunningCount))
```

Comment:
what does default capacity provider mean ?

Reply:
it should be "none" - so 3/5 not scaled by capacity provider